docs/documentation_routes.js

```diff
@@ -51,6 +51,12 @@ router.get('/download', function (req, res) {
   }
 })
 
+router.get('/update.sh', function (req, res) {
```

Comment:
I know this is incredibly unlikely, but are we concerned about what happens if somebody/somebot grabs this page, copies the script and runs it?

It shouldn't do anything unless it's in a prototype folder, correct?

Reply:
> I know this is incredibly unlikely, but are we concerned about what happens if somebody/somebot grabs this page, copies the script and runs it?

I think you could make the same argument about the script that's already on GitHub - we expect that people might try and run the script without fully understanding what they're doing, so we have some precautions, but if it's a case of they are going to go out of their way to run it in unusual circumstances then there's not much one can do to prevent that.

> It shouldn't do anything unless it's in a prototype folder, correct?

Correct. We could also consider having a prompt that asks the user whether they want to continue, although we haven't done any user testing of that.